e2e/node/e2e.test.ts

```diff
@@ -102,186 +96,140 @@ const invalidCredentialClaims = {
-// Instructions for obtaining these credentials can be found here:
-// https://github.com/inrupt/solid-client-authn-js/blob/1a97ef79057941d8ac4dc328fff18333eaaeb5d1/packages/node/example/bootstrappedApp/README.md
-const serversUnderTest: AuthDetails[] = [
-  // Note: Disabled due to PodSpaces 2.0 migration:
-  // pod.inrupt.com:
-
-  // dev-next.inrupt.com:
-  [
-    // Cumbersome workaround, but:
-    // Trim `https://` from the start of these URLs,
-    // so that GitHub Actions doesn't replace them with *** in the logs.
-    process.env.E2E_TEST_DEV_NEXT_IDP_URL!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_VC_SERVICE!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_VC_SUBJECT!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_CLIENT_ID!,
-    process.env.E2E_TEST_DEV_NEXT_CLIENT_SECRET!,
-  ],
-];
-
-describe.each(serversUnderTest)(
-  "VC client end-to-end tests authenticated to [%s], issuing from [%s] for [%s]",
-  (
-    oidcIssuerDisplay,
-    vcServiceDisplay,
-    vcSubjectDisplay,
-    clientId,
-    clientSecret
-  ) => {
-    const oidcIssuer = new URL(`https://${oidcIssuerDisplay}`).href;
-    const vcService = new URL(`https://${vcServiceDisplay}`).href;
-    const vcSubject = new URL(`https://${vcSubjectDisplay}`).href;
-
-    it("has the appropriate environment variables", () => {
-      expect(oidcIssuer).toBeDefined();
-      expect(clientId).toBeDefined();
-      expect(clientSecret).toBeDefined();
-      expect(vcService).toBeDefined();
-      expect(vcSubject).toBeDefined();
-    });
-
-    const session = new Session();
-    // let vcConfiguration: Partial<{
-    //   derivationService: string;
-    //   issuerService: string;
-    //   statusService: string;
-    //   verifierService: string;
-    // }>;
-
-    beforeEach(async () => {
-      await session.login({
-        oidcIssuer,
-        clientId,
-        clientSecret,
-      });
-      // The following code snippet doesn't work in Jest, probably because of
-      // https://github.com/standard-things/esm/issues/706 which seems to be
-      // related to https://github.com/facebook/jest/issues/9430. The JSON-LD
-      // module depends on @digitalbazaar/http-client, which uses esm, which
-      // looks like it confuses Jest. Working around this by hard-coding the
-      // endpoints IRIs.
-      // vcConfiguration = await getVerifiableCredentialApiConfiguration(
-      //   vcService
-      // );
-    });
+const {
+  environment,
+  idp: oidcIssuer,
+  vcProvider,
+  clientCredentials,
+} = getNodeTestingEnvironment({
+  vcProvider: "",
+  clientCredentials: {
+    owner: { id: "", secret: "" },
+  },
+});
 
-    afterEach(async () => {
-      // Making sure the session is logged out prevents tests from hanging due
-      // to the callback refreshing the access token.
-      await session.logout();
+describe(`End-to-end verifiable credentials tests for environment:[${environment}}]`, () => {
```

Comment:
```suggestion
describe(`End-to-end verifiable credentials tests`, () => {
```

The environment is surfaced via the GitHub Actions matrix, no need to log it here.

Reply:
should we remove this for all the other packages that have this printed in the tests then as we go? https://github.com/inrupt/solid-client-authn-js/blob/9680631fb567be865ea1522aa74c3141a5cbd2e3/e2e/node/e2e-test.spec.ts#L38